### ISSUE TITLE
Portfolio view

### ISSUE THREAD

**Issue by fgregg** [datamade/chi-councilmatic#114]
In my estimation, it's pretty hard for an inexperienced and isolated individual to have much effect on matters of important policy before the council.  

An entry point to power can joining and supporting existing organizations focused on your concerns. 

I've been struggling to envision people could get connected to those organizations on our site. Extending @waldoj's "[public portfolio view](https://www.richmondsunlight.com/photosynthesis/6rhd7/)" could be a great way to do this.

Components. 
* Organization users
* Who can track legislation and comment on it
* Profile page for organizations that show how people can get involved and support the *organization* and campaign
* Top level get involved page that lists organizations by topic

I think we could do local outreach to get some early adopting users.

Thoughts @davidmooreppf @derekeder @cathydeng @evz

**Comment by derekeder** [comment]
This was also attempted on Open Congress: https://www.opencongress.org/groups

**Comment by derekeder** [comment]
More details from @davidmooreppf https://www.opencongress.org/articles/view/2344-Announcing-OpenCongress-v3-0-Now-the-Easiest-Way-to-Organize-and-Contact-Congress-/

**Comment by davidmooreppf** [comment]
Agreed @fgregg, those components look right-on to me. The ["verified"](http://www.participatorypolitics.org/become-verified-on-nyc-councilmatic/) public comment program on  NYC Councilmatic aims to lay similar groundwork as OC/OG in '09/'11: convene government offices, community/issue groups, and digital media in a public dialogue. Then as we've been discussing, a native commenting system on Councilmatic will make possible org. profile pages that make the most use of Councilmatic's data. Similarly, our [proposal](http://www.participatorypolitics.org/councilmatic-a-finalist-in-knight-cities-challenge/) for alert features will push out links to comment/track to group membership. So yes, hope to find resources to build this, it would be huge for city-level commenting and surfacing most-relevant information from the haystack.
